libs/net/dal/Migrations/Initial/Up/PostUp/00-Connection.sql

```diff
@@ -67,13 +67,25 @@ INSERT INTO public.connection (
   , 'Meltwater and Blacks News Group upload files to this location.' -- description
   , true -- is_enabled
   , 7 -- connection_type - SSH
-  , '{}' -- configuration
+  , '{"path":"/dsk98/processed/","username":"ckayfish","hostname":"scharnhorst.tno.gov.bc.ca","keyFileName":"id_rsa"}' -- configuration
   , true -- is_read_only
   , 0 -- sort_order
   , DEFAULT_USER_ID
   , ''
   , DEFAULT_USER_ID
   , ''
+),  (
+  'Monitored file location' -- 6
```

Comment:
What is this connection for?  It looks like it is for the destination connection.  But it doesn't identify a path where these files would be kept.  Which means they'll be saved to the root volume folder.

Reply:
This would put the files for each ingest in /data/<code> for example /data/BCNG. Do you want it in a sub-directory?

Reply:
If you don't put it into a sub-directory then this volume `/data` is going to get spammed with folders that are not relevant to most other use-cases.

Reply:
The papers are being copied to /data/papers/<code> now.